stdlib/ballerina-builtin/src/main/ballerina/builtin/tablelib.bal

```diff
@@ -50,6 +52,7 @@ documentation {
 
     T{{dt}} The table object
     P{{func}} The function pointer for delete crieteria
+    R{{}} An `int` the number of deleted record count or `error` if any error occured during removing data
 }
 public native function<table dt> remove(function (any) returns (boolean) func) returns (int|error);
```

Comment:
should we remove the () around union type returns and use error? instead of error|() ?